roles/kubernetes/tasks/master.yml

```diff
@@ -2,7 +2,7 @@
 
 - name: Check if the infrastructure is multi-master
   set_fact:
-    k8s_multi_master: "{{ ansible_play_batch | map('extract', hostvars, 'k8s_role') | select('equalto', 'master') | length }}"
+    k8s_multi_master: "{{ ansible_play_batch | map('extract', hostvars, 'k8s_role') | select('equalto', 'master') | length > 1 }}"
```

Comment:
I need to understand what we're doing here, since the k8s_multi_master variable is crucial for the cluster configuration.
My suggestion is to make k8s_multi_master a boolean defaulted to false. If the user wants to do a clustered configuration, then an override will be created, so please remove the usage of ansible_play_patch here, which is pure over complexity.
Less is more, and instead of counting the number of declared masters we want the user to set:
```yaml
k8s_multi_master: true
```
And we're good!